Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -631,6 +632,21 @@ vector<EventsSearchResult> EventsRefactorer::SearchInEvents(
       }
     }
 
+    if (inComments) {
+      vector<gd::String> stringsVectors = events[i].GetAllSearchableStrings();

```

Comment:
This line is correct.

Explanation:
So here you have a vector of string (i.e: a list of strings) that represents "all the strings that can be searched in events number i".
So now your goal is too.. search in these strings. If you find what is searched, then good, you add the event to the results :) Otherwise you don't do anything.